Enformer.pm

```diff
@@ -0,0 +1,220 @@
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=head1 CONTACT
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+=cut
+
+=head1 NAME
+ Enformer
+
+=head1 SYNOPSIS
+ 
+ mv Enformer.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin Enformer,file=Enformer_grch38.vcf.gz
+
+
+=head1 DESCRIPTION
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that adds pre-calculated scores from Enformer.
+
+ This Plugin is available for GRCh37 and GRCh38
+ 
+ Please cite the Enformer publication alongside the VEP if you use this resource:
+ https://www.nature.com/articles/s41592-021-01252-x
+
+ Enformer scores can be downloaded from https://ftp.ensembl.org/pub/current_variation/Enformer for GRCh37 and GRCh38.
+
+ The plugin can then be run as default:
+ ./vep -i variations.vcf --assembly GRCh38 --plugin Enformer,file=/path/to/Enformer/data.vcf.gz
+
+ or run with option to only retrieve the SAD (SNP Activity Difference (SAD) scores - main variant effect score computed as model(alternate_sequence) - model(reference_sequence) score 
+ ./vep -i variations.vcf --assembly GRCh38 --plugin Enformer,file=/path/to/Enformer/data.vcf.gz,SAD=1 
+ 
+ or run with option to only retrieve the SAR (Same as SAD, by computing np.log2(1 + model(alternate_sequence)) - np.log2(1 + model(reference_sequence)) score 
+ ./vep -i variations.vcf --assembly GRCh38 --plugin Enformer,file=/path/to/Enformer/data.vcf.gz,SAR=1 
+
+ or run with option to only retrieve the principal component scores which are a reduced representation of a much bigger vector of the SAD and SAR after using principal component analysis (PCA)
```

Comment:
We normally let people choose to report multiple values together; what is the reason for either/or set up for PCs?

Reply:
Just to clarify, do we want people to report SAD, SAR and PC scores together? 
Thank you